cmd/release-controller/types.go

```diff
@@ -466,6 +466,9 @@ const (
 	// releaseAnnotationBugsVerified indicates whether or not the release has been
 	// processed by the BugzillaVerifier
 	releaseAnnotationBugsVerified = "release.openshift.io/bugs-verified"
+
+	// releaseAnnotationSoftDelete indicates automation external to the release controller can use this annotation to decide when to clean up the tag
```

Comment:
Describe in this comment that the value is a timestamp (3339 as per the above comment if this is a net new addition)

Reply:
Updated.